build/images/scripts/restart_sandbox

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+CRICTL_COMMAND="crictl --runtime-endpoint unix:///run/containerd/containerd.sock"
+
+if [ -f /etc/cni/net.d/10-antrea.conflist ]; then
```

Comment:
antrea-agent tries to ensure the file exist in its own script, moving it to another name may lead to divergence when later antrea-agent updates the conf file, like appending another 3rd plugin.
I feel it could instead rename all conf before this one, for example, append ".bak" to them. I assume container runtime will ignore them then.